eva-server/src/main/java/uk/ac/ebi/eva/server/ws/VariantWSServerV2.java

```diff
@@ -51,22 +53,20 @@
 @RestController
 @RequestMapping(value = "/v2/variants", produces = "application/json")
 @Api(tags = {"variants"})
-public class VariantWSServerV2 extends EvaWSServer {
+public class VariantWSServerV2  {
 
     @Autowired
     private VariantWithSamplesAndAnnotationsService service;
 
     @GetMapping(value = "/{variantCoreString}")
-    public Resource getCoreInfo(@PathVariable("variantCoreString") String variantCoreString,
```

Comment:
I thought we were going to use hateoas in all the v2 endpoints, weren't we?

Reply:
Yes, I am returning 
` return new ResponseEntity(new Resource<>(variant, links),HttpStatus.OK);`
So the output format would still be the same.
Use of ResponseEntity would help to return the cases where error happens easily, I guess